internal/instanceID.go

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Mainflux
```

Comment:
No need to have a separate file. You can have this in `/internal/env/parser.go`. This also means that we don't need  this:
```go
	//create an instanceID for the service
	instanceID, err := uuid.New().ID()
	if err != nil {
		log.Fatalf("instance ID not generated: %s", err)
	}
```
in `main.go` and we'll use config.InstanceID instead.

Reply:
We'll have something like:
```go
func main() {
	ctx, cancel := context.WithCancel(context.Background())
	g, ctx := errgroup.WithContext(ctx)

	c := config{}
	if err := env.Parse(&c); err != nil {
		log.Fatalf("failed to load %s configuration : %s", svcName, err)
	}

	logger, err := mflog.New(os.Stdout, c.LogLevel)
	if err != nil {
		log.Fatalf("failed to init logger: %s", err)
	}

	cfg := server.Config{}
	if err := env.Parse(&cfg, env.Options{Prefix: envPrefix}); err != nil {
		log.Fatalf("failed to load %s server configuration: %s", svcName, err)
	}

	tc, tcHandler, err := thingsClient.Setup(envPrefix, cfg.TracingURL)
	if err != nil {
		logger.Fatal(err.Error())
	}
	defer tcHandler.Close()
	logger.Info("Successfully connected to things grpc server " + tcHandler.Secure())

	tracer, closer, err := jaegerClient.NewTracer(svcName, cfg.TracingURL, cfg.InstanceID)
	if err != nil {
...
```